### ISSUE TITLE
installation problems on CentOS 6.6 64 bit

### ISSUE THREAD

**Issue by wmsvision** [odeke-em/drive#329]
Hello,
      I ran this command on a CentOS machine: `go get -u github.com/odeke-em/drive/cmd/drive`.  I keep getting this error: `go build golang.org/x/text/collate: /usr/lib/golang/pkg/tool/linux_amd64/6g: signal: killed`.  Why do I get this error?

Some other information:
```
+ lsb_release -a
LSB Version:    :base-4.0-amd64:base-4.0-noarch:core-4.0-amd64:core-4.0-noarch
Distributor ID: CentOS
Description:    CentOS release 6.6 (Final)
Release:        6.6
Codename:       Final
+ echo

+ echo 'GOPATH = /usr/lib/golang/'
GOPATH = /usr/lib/golang/
+ echo

+ whereis go
go: /usr/bin/go
+ echo

+ go version
go version go1.4.2 linux/amd64
```

Tell me if you need more information.  Thanks.

**Comment by odeke-em** [comment]
Hello @wmsvision, thanks for reporting this issue and my apologies for the late reply.

So the cause of this was that golang's text/collate package was consuming excessive amounts of memory. I've ripped it out since the collator being used was even using an undefined language locale.

Please get the latest from master by
```shell
$ go get -u github.com/odeke-em/drive/drive-gen && drive-gen
```

**Comment by odeke-em** [comment]
Addressed by https://github.com/odeke-em/drive/pull/331

**Comment by wmsvision** [comment]
Thanks.  I tried what you suggested and now I get a new problem.  Should I add a new issue?

I got this response when I tried the exact command you gave me:
```
[root@ip-172-31-27-149 ~]# go get -u github.com/odeke-em/drive/drive-gen && drive-gen
# cd /usr/lib/golang/src/github.com/odeke-em/drive; git pull --ff-only
From https://github.com/odeke-em/drive
 + 4395dee...bae47eb master     -> origin/master  (forced update)
 * [new branch]      sort-no-collator -> origin/sort-no-collator
fatal: Not possible to fast-forward, aborting.
package github.com/odeke-em/drive/drive-gen: exit status 128
[root@ip-172-31-27-149 ~]# 
```
I then tried the following variations on your command:
```
[root@ip-172-31-27-149 ~]# go get -u github.com/odeke-em/drive/drive-gen
# cd /usr/lib/golang/src/github.com/odeke-em/drive; git pull --ff-only
fatal: Not possible to fast-forward, aborting.
package github.com/odeke-em/drive/drive-gen: exit status 128
[root@ip-172-31-27-149 ~]# 

[root@ip-172-31-27-149 ~]# go get -u github.com/odeke-em/drive
# cd /usr/lib/golang/src/github.com/odeke-em/drive; git pull --ff-only
fatal: Not possible to fast-forward, aborting.
package github.com/odeke-em/drive: exit status 128

[root@ip-172-31-27-149 ~]# go get -u github.com/odeke-em/drive/cmd/drive
# cd /usr/lib/golang/src/github.com/odeke-em/drive; git pull --ff-only
fatal: Not possible to fast-forward, aborting.
package github.com/odeke-em/drive/cmd/drive: exit status 128
[root@ip-172-31-27-149 ~]# 
```
Am I doing something wrong?

**Comment by odeke-em** [comment]
Hello @wmsvision seems more of a git problem than with drive. The easiest way would be to go into your $GOPATH/src (if it is properly setup) and then `rm -rf github.com/odeke-em` ie
```shell
$ rm -rf $GOPATH/src/github.com/odeke-em
$ go get -u github.com/odeke-em/drive/drive-gen && drive-gen
```

**Comment by wmsvision** [comment]
Thanks.  That worked, once I added "$GOPATH/bin" to my PATH environmental variable.

In my case, whether right or wrong, I also took the liberty of making softlinks from "$GOPATH/bin" to /usr/bin, as follows:
```
[root@ip-172-31-27-149 ~]# ls -lt /usr/bin/*drive*
lrwxrwxrwx. 1 root root 29 Aug  7 00:00 /usr/bin/drive-gen -> /usr/lib/golang/bin/drive-gen
lrwxrwxrwx. 1 root root 25 Aug  7 00:00 /usr/bin/drive -> /usr/lib/golang/bin/drive
[root@ip-172-31-27-149 ~]# 
```
Not really needed if we added "$GOPATH/bin" to PATH for all users, but I did not want to to change PATH, so just made softlinks.  Obviously, this is totally a personal choice, not needed to use  your t ool at all, just mentioning it in case others want to know.
